biz.aQute.bndlib/src/aQute/bnd/osgi/Analyzer.java

```diff
@@ -247,7 +247,13 @@ public void analyze() throws Exception {
 					// built ins
 					//
 
-					cds.add(annotationHeaders = new AnnotationHeaders(this));
+					if (since(About._4_2)) {
```

Comment:
Do we really need the `since`? We are adding a feature to do less than everything. And the default is everything. So I don't see any backward compatibility here that may need to be switched off.